go.sum

```diff
@@ -1,5 +1,6 @@
 github.com/PuerkitoBio/goquery v1.5.0 h1:uGvmFXOA73IKluu/F84Xd1tt/z07GYm8X49XKHP7EJk=
 github.com/PuerkitoBio/goquery v1.5.0/go.mod h1:qD2PgZ9lccMbQlc7eEOjaeRlFQON7xY8kdmcsrnKqMg=
+github.com/akellbl4/WoD v0.0.0-20200111172559-91e8e67aee77 h1:+aNnCtxjtItv5YgFvMeAQ0cCoXGQ0yA0g7UICEwvrBs=
```

Comment:
Something is not right, it should not be there. I suggest to try `go mod tidy`, if it won't help at first - with deleting go.sum and trying again.